run

```diff
@@ -0,0 +1,82 @@
+# A simple setup for convection in a quarter of a 2d shell. See the
```

Comment:
You add a file called `run` in the global directory. I don't think that's what you wanted to do, or at least probably not here.